sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -4,13 +4,23 @@
 
 ### Features Added
 
+- Added `GetIv()` to `EncryptParameters` and `DecryptParameters`.
+
 ### Breaking Changes
 
+- Removed `Azure::Security::KeyVault::Keys::ServiceVersion::V7_0`.
+- Removed `Azure::Security::KeyVault::Keys::ServiceVersion::V7_1`.
+- Removed `Azure::Security::KeyVault::Keys::Cryptography::ServiceVersion::V7_0`.
+- Removed `Azure::Security::KeyVault::Keys::Cryptography::ServiceVersion::V7_1`.
+- Removed `CryptographyClient::RemoteClient()`.
+- Removed `CryptographyClient::LocalOnly()`.
+- Removed the general constructor from `EncryptParameters` and `DecryptParameters`.
```

Comment:
What's a general constructor?

Reply:
a cttor which takes all possible init values for the class.  I though it would take a very long text to add the entire signtature